src/wallet/reconstruct_seed_flow.c

```diff
@@ -153,10 +153,10 @@ reconstruct_state_e reconstruct_seed_handler(reconstruct_state_e state,
       ui_scrollable_page(
           ui_text_confirm_passphrase, display, MENU_SCROLL_HORIZONTAL, false);
       memzero(display, sizeof(display));
-      next_state = get_state_on_confirm_scr(
-          PASSPHRASE_CONFIRM, PASSPHRASE_INPUT, TIMED_OUT);
+      next_state =
+          get_state_on_confirm_scr(PIN_INPUT, PASSPHRASE_INPUT, TIMED_OUT);
 
-      if (PASSPHRASE_CONFIRM == next_state) {
+      if (PIN_INPUT == next_state) {
```

Comment:
would be best if we split this fix as a separate PR

Reply:
raised separate [PR](https://github.com/Cypherock/x1_wallet_firmware/pull/334)